config/dev.cfg

```diff
@@ -16,7 +16,8 @@ credential.password=testPwd
 
 [path]
 path.ebooks=ebooks
-path.extras=ebooks/extras
+path.extras=extras
+path.separate=true
```

Comment:
That's a good idea. I also thought if true/false is actually needed. This can be changed. ``path.separate.ebooks`` and ``path.separate.extras`` is also a good idea. I will try to change this in the near future.